ql/src/test/queries/clientpositive/vectorized_murmurhash_repeating.q

```diff
@@ -0,0 +1,29 @@
+set hive.llap.io.enabled=false;
+set hive.vectorized.execution.enabled=true;
+
+CREATE TABLE source(id string, id2 string) stored as orc;
+LOAD DATA LOCAL INPATH '../../data/files/2048.orc' overwrite into table source;
```

Comment:
The .orc are binary files that cannot be easily inspected/reviewed. Moreover, ASF imposes various limitations on the content of binary files so whenever we can avoid them it is good.
Is it possible to use a .csv file as an input and use a staging TEXT table for loading the data to the ORC table? If it's not possible then that's fine we can merge the PR as is.

Reply:
Sure we can. I changed qfile to create ORC table from text file and removed ORC file.